pkg/apis/extensions/validation/operatingsystemconfig.go

```diff
@@ -77,6 +80,103 @@ func ValidateOperatingSystemConfigStatus(status *extensionsv1alpha1.OperatingSys
 	return allErrs
 }
 
+// ValidateCriConfig validates the spec of a CRIConfig object.
+func ValidateCriConfig(config *extensionsv1alpha1.CRIConfig, purpose extensionsv1alpha1.OperatingSystemConfigPurpose, fldPath *field.Path) field.ErrorList {
```

Comment:
Nit

```suggestion
func ValidateCRIConfig(config *extensionsv1alpha1.CRIConfig, purpose extensionsv1alpha1.OperatingSystemConfigPurpose, fldPath *field.Path) field.ErrorList {
```

as the field/type itself is `CRIConfig`.